source/extensions/path/pattern_template_lib/pattern_template_internal.h

```diff
@@ -0,0 +1,147 @@
+#pragma once
+
+#include <optional>
+#include <string>
+#include <variant>
+#include <vector>
+
+#include "absl/container/flat_hash_set.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "absl/types/optional.h"
+#include "absl/types/variant.h"
+#include "re2/re2.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace PatternTemplate {
+
+namespace Internal {
+
+using Literal = absl::string_view;
```

Comment:
nit: Comment, please. (though it's reasonable readable as is)

Reply:
Added